internal/util/reftracker/errors/errors.go

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2022 The Ceph-CSI Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package errors
+
+import (
+	goerrors "errors"
+	"fmt"
+
+	"golang.org/x/sys/unix"
+
+	"github.com/ceph/go-ceph/rados"
```

Comment:
no new line is required as both are third party https://github.com/ceph/ceph-csi/blob/devel/internal/util/util.go#L31